docs/tutorial/advanced/mkda_association.md

```diff
@@ -0,0 +1,92 @@
+
+See "Further Reading" below to read more about why this change was made
+:::
+
+## Specifying MKDA Chi-Squared Meta-Analysis
+
+Specifying an MKDA Chi-Square meta-analysis in Neurosynth is easy. Simply, select a target set of studies as you would for any other meta-analysis, using either automated or manual selection methods. 
+
+In Step 3 ("Create Meta-Analysis Specification") of your Project, select *MKDAChi2* as the *algorithm*. 
+
+
+![MKDA Chi Squared](/tutorial/mkda_chi_squared_algo.png). 
+
+Next, select the annotation inclusion column you want to use, as before (by default, the "included" column will be used).
+
+Now, select a reference dataset from the dropdown list below. The Neurosynth dataset represents the latest release of the legacy *Neurosynth* dataset (version 7), released July, 2018. The *Neurostore* dataset represents the latest update of our continuously updating "live" dataset, spanning over 20,000 neuroimaging studies. 
+
+![MKDA Chi Squared Reference](/tutorial/mkda_chi_squared_reference.png). 
+
+Now simply complete the rest of the meta-analysis specification wizard to finish. 
+
+## Executing your analysis
+
+The *MKDA Chi-Squared* algorithm is more computational expensive than a traditional *MKDA Density* analysis. As a result, it's unlikely to complete with the freely available resources available on Google Colab.
+
+You can run this workflow locally in one line using docker by copying the command your screen. See the *execution* documentation page for more information. 
+
+:::tip 
+We recommend at least 32GB of RAM to perform a MKDA Chi-Squared analysis
+:::
+
+## Interpreting results
+
+![Neurosynth Maps](/tutorial/neurosynth_paper_fig2.jpg). 
+
+**Figure 1**. Comparison of Uniformity and Association maps from the original Neurosynth article, on three automatically generated meta-analyses.
+
+The *MKDA Chi-Squared* Workflow outputs two types of maps: **uniformity** and **association** test maps.
+
+- **Uniformity test map:** z-scores from a one-way ANOVA testing whether the proportion of studies that report activation at a given voxel differs from the rate that would be expected if activations were uniformly distributed throughout gray matter.
+
+The uniformity test map can be interpreted in roughly the same way as most standard whole-brain fMRI analysis: it displays the degree to which each voxel is consistently activated in studies that use a given term. For instance, for a meta-analysis of "emotion" high z-scores in the amygdala implies that studies that use the word emotion a lot tend to consistently report activation in the amygdala--at least, more consistently than one would expect if activation were uniformly distributed throughout gray matter. Note that, unlike other coordinate based meta-analysis algorithms (e.g., ALE or MKDA), z-scores aren't generated through permutation, but using a chi-square test
```

Comment:
I believe NiMARE does support multiple comparisons correction for MKDAChi2 at this point, so corrected z-scores can be calculated.